paper.qmd

```diff
@@ -597,6 +604,53 @@ knitr::include_graphics("images/fig5.png")
 
 ::: -->
 ```
+## Stakeholder engagement {#sec-stakeholder-engagement}
+
+The CRUSE Tool was developed for, and in close collaboration with, practitioners in Ireland.
+Although the overall approach of the project was quantitative, stakeholder feedback and qualitative data was deemed essential to ensure the tool was fit for purpose, mitigating against the risk that the project would be 'technocentric' and a 'problem looking for a solution'.
+From the outset of the project, we aimed to follow @coppola2022's advocacy for a "human-centric approach in which personalization, purpose, ethics and inclusivity are key", rather than an 'off-the-shelf' product that failed to understand, let alone meet, local needs.
+As outlined in the overview of papers in the collection Inclusive and Collaborative Transport cited above, the 'digital transformation' taking place in transport planning (and many other sectors) has the potential to increase existing inequalities [@coppola2022].
+A wide range of methods and measure can be taken to ensure that digital tools are inclusive \[...\].
```

Comment:
I'd remove the three dots and perhaps change the term "inclusive" to "are accessible to their recipients.